rules/azure/persistence_azure_automation_webhook_created.toml

```diff
@@ -0,0 +1,36 @@
+[metadata]
+creation_date = "2020/08/18"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when an Azure Automation webhook is created. Azure Automation runbooks can be configured to execute via a
+webhook. A webhook is a custom URL passed to Azure Automation along with a data payload specific to the runbook. An
+adversary may create a webhook in order to trigger a runbook that contains malicious code.
+"""
+from = "now-25m"
+index = ["filebeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Azure Automation Webhook Created"
+note = "The Azure Filebeat module must be enabled to use this rule."
+references = [
+    "https://powerzure.readthedocs.io/en/latest/Functions/operational.html#create-backdoor",
+    "https://github.com/hausec/PowerZure",
+    "https://posts.specterops.io/attacking-azure-azure-ad-and-introducing-powerzure-ca70b330511a",
+    "https://www.ciraltos.com/webhooks-and-azure-automation-runbooks/",
+]
+risk_score = 21
+rule_id = "e9ff9c1c-fe36-4d0d-b3fd-9e0bf4853a62"
+severity = "low"
+tags = ["Elastic", "Azure", "Continuous Monitoring", "SecOps", "Configuration Audit"]
+to = "now-25m"
+type = "query"
+
+query = '''
+event.module:azure and event.dataset:azure.activitylogs and event.category:Administrative and azure.activitylogs.operation_name:(MICROSOFT.AUTOMATION/AUTOMATIONACCOUNTS/WEBHOOKS/ACTION or MICROSOFT.AUTOMATION/AUTOMATIONACCOUNTS/WEBHOOKS/WRITE) and event.outcome:Success
```

Comment:
does `MICROSOFT.AUTOMATION/AUTOMATIONACCOUNTS/WEBHOOKS/ACTION` log everytime a webhook is executed or only when it is created?

Reply:
Good question. I just reviewed the logs again and confirmed that no event was logged when I actually triggered the webhook.

Reply:
Awesome, thanks